core/src/main/java/com/linecorp/armeria/server/TransientService.java

```diff
@@ -31,6 +36,17 @@
 @FunctionalInterface
 public interface TransientService<I extends Request, O extends Response> extends Service<I, O> {
 
+    /**
+     * Returns a new {@link HttpService} decorator which makes the specified {@link HttpService} as
+     * {@link TransientService}.
+     */
+    static Function<? super HttpService, SimpleDecoratingHttpService> newDecorator(
```

Comment:
Should we move this to `TransientHttpService`?

Reply:
Ah... you are right. I thought this is an `HttpService` like `LoggingService`. 😅